src/main/java/fr/openent/jupyter/Jupyter.java

```diff
@@ -37,6 +38,9 @@ public void start() throws Exception {
 		addController(new DirectoryController(eb, storage));
 		addController(new FileController(eb, storage));
 		addController(new ManagerController(eb, storage));
+
+		startPromise.tryComplete();
+		startPromise.tryFail("[Jupyter-connector@Jupyter-connector::start] Fail to start Jupyter-connector");
```

Comment:
Les logs dans la forme c'est "[Nomdumodule@Nomdelaclasse::nomdelafonction]"
Donc ici t'auras plus :
```suggestion
		startPromise.tryFail("[Jupyter-connector@Jupyter::start] Fail to start Jupyter-connector");
```